README.md

```diff
@@ -79,38 +349,73 @@ The same model can be trained via command line:
 DeezyMatch -i ./inputs/input_dfm.yaml -d dataset/dataset-string-similarity_test.txt -m test001
 ```
 
-A new model directory called `test001` will be created in `models` directory (as specified in the `models_dir` in the input file).
+---
+
+Summary of the arguments/flags:
+
+| Func. argument   	| Command-line flag 	| Description                                       	|
+|------------------	|-------------------	|---------------------------------------------------	|
+| input_file_path  	| -i                	| path to the input file                            	|
+| dataset_path     	| -d                	| path to the dataset                               	|
+| model_name       	| -m                	| name of the new model                             	|
+| n_train_examples 	| -n                	| number of training examples to be used (optional) 	|
+
+---
+
+A new model directory called `test001` will be created in `models` directory (as specified in the input file, see `models_dir` in `./inputs/input_dfm.yaml`).
 
 :warning: Dataset (e.g., `dataset/dataset-string-similarity_test.txt` in the above command)
 * Currently, the third column (label column) should be one of (case-insensitive): ["true", "false", "1", "0"]
 * Delimiter is fixed to `\t` for now.
 
 DeezyMatch keeps some information about the metrics (e.g., loss/accuracy/precision/recall/F1) for each epoch. It is possible to plot the log-file by:
 
+```python
+from DeezyMatch import plot_log
+
+# plot log file
+plot_log(path2log="./models/test001/log.txt", 
+         output_name="t001")
+```
+
+or:
+
 ```bash
-DeezyMatch -lp ./models/test001/log.txt -ld test001
+DeezyMatch -lp ./models/test001/log.txt -lo t001
 ```
 
-In this command, 
-* `-lp`: runs the log plotter
-* `-ld` is a name assigned to the log which will be used in the figure. 
+---
+
+Summary of the arguments/flags:
+
+| Func. argument 	| Command-line flag 	| Description                                  	|
+|----------------	|-------------------	|----------------------------------------------	|
+| path2log       	| -lp               	| path to the log file                         	|
+| output_name    	| -lo               	| output name (normally, name of the dataset). 	|
```

Comment:
Not sure about "(normally, name of the dataset)". It may be more confusing than helpful (the dataset used to train the original model? the dataset used for fine-tuning?). Since this output file is already stored inside the model folder, I think we probably can remove the text in the parentheses.

Reply:
Done: https://github.com/Living-with-machines/DeezyMatch/pull/68/commits/414b9f1f13da078fdbcf5a376f16063192cddca1